markdoc/editor/hotkeys/hotkeys.markdoc.ts

```
@@ -54,107 +54,127 @@ export const useMarkdownHotkeys = (
-        const cursorPos =
-          type === "wrap" && hotkey.key === "c" && hotkey.useShift
-            ? startPos + markup.length + 1
-            : startPos + newText.length;
-        textarea.setSelectionRange(cursorPos, cursorPos);
+              endPos,
+            )}${markup}`;
+          }
+          break;
+
+        case "blockQuote":
+          const lines = currentValue.slice(startPos, endPos).split("\n");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)